core/src/main/java/bio/terra/pearl/core/model/fileupload/ParticipantFileUpload.java

```diff
@@ -0,0 +1,27 @@
+package bio.terra.pearl.core.model.fileupload;
+
+import bio.terra.pearl.core.model.BaseEntity;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.SuperBuilder;
+
+import java.util.UUID;
+
+@Getter
+@Setter
+@SuperBuilder
+@NoArgsConstructor
+public class ParticipantFileUpload extends BaseEntity {
+    private String fileName;
+    private String fileType; // e.g. pdf, docx, etc.
+
+    private UUID uploadedFileId; // files are uploaded as UUIDs
```

Comment:
```suggestion
    private UUID externalFileId; // files are uploaded as UUIDs
```

Reply:
`uploaded` is ambiguous here, so I'd go with `external` or `backend`.  I could also see `externalFileName` since it's just a name in the external system